browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -282,8 +282,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			// returns docnotloaded error. Instead of this we can return to the integration
 			if (!builder.map._docLoaded &&
 				 !window._firstDialogHandled &&
-				 ((object.id === 'cancel' || eventType === 'close') ||
-				 (objectType === 'responsebutton' && (data == 0 || data == 7)))) {
+				 eventType === 'close') {
```

Comment:
please also handle `responsebutton` case, it's present in some cases.
`-7` is `close` I think: https://opengrok.libreoffice.org/xref/core/vcl/source/window/builder.cxx?r=174430d7#set_response